From 657e2e47113d84c5eff2105cced31d5f66e39458 Mon Sep 17 00:00:00 2001
From: Malte Moeller <mm@freifunk-aachen.de>
Date: Fri, 18 Jan 2019 23:06:55 +0100
Subject: [PATCH] change device identifier to match machine identifier this
 matches the output of lua -e 'print(require(platform_info).get_image_name())'

This way the manifest file is accepted by the autoupdater
---
 targets/ar71xx-generic | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/targets/ar71xx-generic b/targets/ar71xx-generic
index 96b19d8..2f6828b 100644
--- a/targets/ar71xx-generic
+++ b/targets/ar71xx-generic
@@ -79,7 +79,7 @@ factory
 device gl-ar300m gl-ar300m
 factory
 
-device gl-ar750 gl-ar750
+device gl.inet-gl-ar750 gl-ar750
 packages $ATH10K_PACKAGES
 factory
 
-- 
2.5.0

